universe/Fleet.h

```diff
@@ -140,17 +140,7 @@ class FO_COMMON_API Fleet : public UniverseObject {
     friend class ObjectMap;
 
     /** \name Structors */ //@{
-    Fleet() :
-        UniverseObject(),
-        m_prev_system(INVALID_OBJECT_ID),
-        m_next_system(INVALID_OBJECT_ID),
-        m_aggressive(true),
-        m_ordered_given_to_empire_id(ALL_EMPIRES),
-        m_travel_route(),
-        m_travel_distance(0.0),
-        m_arrived_this_turn(false),
-        m_arrival_starlane(INVALID_OBJECT_ID)
-    {}
+    Fleet() {}
```

Comment:
Should I still do this?
I figured the (local) style was for 1 line.
The public destructor has {} on the same line, and a little further up there are functions like SetArrivalStarlane that have the body on the same line.